apps/docs/pages/guides/functions/deploy.mdx

```diff
@@ -48,6 +48,13 @@ supabase link --project-ref your-project-id
 
 ## Deploy your Edge Functions
 
+<Admonition type="tip" label="Docker required">
+
+Since Supabase CLI version 1.123.4, you must have a [Docker client](https://docs.docker.com/desktop/) installed in your machine to deploy Edge Functions.
```

Comment:
```suggestion
Since Supabase CLI version 1.123.4, you must have a [Docker Desktop](https://docs.docker.com/desktop/) installed in your machine to deploy Edge Functions.
```

nitpick: the term we used in CLI installation guide is Docker Desktop (to align with mac and windows clients). It's probably worth using the same name for consistency.